lib/smart_proxy_remote_execution_ssh/runners/polling_script_runner.rb

```diff
@@ -126,7 +125,11 @@ def load_event_updates(event_data)
     end
 
     def cleanup
-      run_sync("rm -rf \"#{remote_command_dir}\"") if @cleanup_working_dirs
+      if @cleanup_working_dirs
+        assert_remote_command("rm -rf \"#{remote_command_dir}\"",
```

Comment:
what is `assert_remote_command`? assert should not run anything in my mind... it should just checks if something had run (had happend)...

Reply:
Right, it is "run the thing, check if it succeeded and raise an exception if not". If you thing `ensure_*` would depict it better then I'll change it

Reply:
I think so, it is not great, but IMO bit better then assure